openpype/hosts/maya/api/commands.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+"""OpenPype script commands to be used directly in Maya."""
+import sys
+
+
+def edit_shader_definitions():
+    from avalon.tools import lib
+    from Qt import QtWidgets
+    from openpype.hosts.maya.api.shader_definition_editor import (
+        ShaderDefinitionsEditor
+    )
+
+    module = sys.modules[__name__]
+    module.window = None
```

Comment:
Missing validation of existing window (I can create as many windows as I like to). If it's in plan to store more than one "tool" here I would recommend to have class with classmethods that will care about storing the tool windows instead of using global variable `window`...

![image](https://user-images.githubusercontent.com/43494761/123606625-5eac9b00-d7fd-11eb-81ac-ae538fb74424.png)